packages/devtools_app/lib/src/inspector/inspector_breadcrumbs.dart

```diff
@@ -129,41 +135,62 @@ class _InspectorBreadcrumb extends StatelessWidget {
 }
 
 class _InspectorBreadcrumbData {
-  const _InspectorBreadcrumbData._(
-    this.row,
-    this.isSelected,
-    this._alternativeText,
-    this._alternativeIcon,
-  );
-
-  factory _InspectorBreadcrumbData.wrap(
-          InspectorTreeRow row, bool isSelected) =>
-      _InspectorBreadcrumbData._(row, isSelected, null, null);
+  const _InspectorBreadcrumbData._({
+    @required this.row,
+    @required this.isSelected,
+    @required this.alternativeText,
+    @required this.alternativeIcon,
+  });
+
+  factory _InspectorBreadcrumbData.wrap({
+    @required InspectorTreeRow row,
+    @required bool isSelected,
+  }) {
+    return _InspectorBreadcrumbData._(
+      row: row,
+      isSelected: isSelected,
+      alternativeText: null,
+      alternativeIcon: null,
+    );
+  }
 
   /// Construct a special item for showing '…' symbol between other items
-  factory _InspectorBreadcrumbData.more() =>
-      const _InspectorBreadcrumbData._(null, false, '…', null);
+  factory _InspectorBreadcrumbData.more() {
+    return const _InspectorBreadcrumbData._(
+      row: null,
+      isSelected: false,
+      alternativeText: '…',
+      alternativeIcon: null,
+    );
+  }
 
   factory _InspectorBreadcrumbData.chevron() {
     final icon = Icon(
       Icons.chevron_right,
       size: defaultIconSize,
     );
-    return _InspectorBreadcrumbData._(null, false, null, icon);
+    return _InspectorBreadcrumbData._(
+      row: null,
+      isSelected: false,
+      alternativeText: null,
+      alternativeIcon: icon,
+    );
   }
 
   final InspectorTreeRow row;
-  final Widget _alternativeIcon;
-  final String _alternativeText;
+  final Widget alternativeIcon;
+  final String alternativeText;
   final bool isSelected;
 
-  String get text => _alternativeText ?? row?.node?.diagnostic?.description;
+  String get text => alternativeText ?? row?.node?.diagnostic?.description;
 
-  Widget get icon => _alternativeIcon ?? row?.node?.diagnostic?.icon;
+  Widget get icon => alternativeIcon ?? row?.node?.diagnostic?.icon;
 
-  bool get isChevron => row == null && _alternativeText == null;
+  bool get isChevron =>
```

Comment:
why not just check if `row == null && alternativeIcon == Icons.chevron_right`. Same for isEllipsis: `row == null && alternativeText == _ellipsisText` where _ellipsisText is a const value pulled out and used both here and in the .more constructor